packages/office-ui-fabric-react/src/components/ChoiceGroup/examples/ChoiceGroup.Basic.Example.tsx

```diff
@@ -26,7 +26,12 @@ export class ChoiceGroupBasicExample extends React.Component<any, IChoiceGroupBa
           options={ [
             {
               key: 'A',
-              text: 'Option A'
+              text: 'Option A',
+              onRenderField: (props, render) => {
```

Comment:
Could we add another example, so that the basic example stays as basic as possible?